src/StructuredLogViewer/Controls/BuildControl.xaml.cs

```diff
@@ -2056,6 +2056,33 @@ public void DisplayStats()
             //statsRoot.AddChild(histogramNode);
 
             statsRoot.AddChild(new Property { Name = "BinlogFileFormatVersion", Value = Build.FileFormatVersion.ToString() });
+            if (Build.IsCompatibilityMode)

```

Comment:
This is nice but perhaps too much detail? Would be certainly useful to us, but probably too much detail for our users.

Since we already have it, I'm OK with keeping it, just extract method here.